core/config/docs/chains-tron.toml

```diff
@@ -0,0 +1,13 @@
+[[Tron]]
+# ChainID is the Tron chain ID.
+ChainID = 'foobar' # Example
+# FeederURL is required to get tx metadata (that the RPC can't)
+FeederURL = 'http://feeder.url' # Example
```

Comment:
Will there always just be one of these? Or should we consider supporting multiple as a list or something?